doc/solver/equation_object.rst

```diff
@@ -0,0 +1,19 @@
+.. _equation-object-class:
+
+Equation object class
++++++++++++++++++++++++++++++++++++++++++++++
+An equation object consists of three parts:
+
+#. A list of the index in the matrix of the coefficients of the equation
+#. A list of the coefficients of the equation
+#. The right hand side of the equation
+
+The index is the sum of the index in the matrix, which is obtained by the assets by setting their unknowns in the matrix with the quantity to be solved.
+This can be easily obtained by making use of the CoreQuantIndex class.
+The coefficients are the coefficients which are obtained by linearizing the equations. The same holds for the right hand side. This linearization should be done via the Newton-Raphson methode.
+More information can be found on this `website`_.
+
+.. autoclass:: simulator_core.solver.matrix.equation_object.EquationObject
+   :members:
+
+.. _website: https://en.wikipedia.org/wiki/Newton%27s_method
```

Comment:
I think something went wrong with formatting?  (link is separate from text).   Or is this a reference to a footnote/reference list?

Reply:
In this way the text in the html becomes website and not the full website path. So it is better readable
![image](https://github.com/user-attachments/assets/7b3470a4-f1cd-469a-9efa-00a04e96255e)